src/test/java/com/snowflake/kafka/connector/ConnectorConfigTest.java

```diff
@@ -7,6 +7,7 @@
 import static com.snowflake.kafka.connector.internal.TestUtils.getConfig;
 import static org.junit.Assert.assertEquals;
 
+import com.snowflake.kafka.connector.internal.OAuthConstants;
```

Comment:
overall I am seeing great value addition for all types of tests. 
I dont see End to End tests though, do you plan to add it?

Reply:
Discussed offline. Will added in another PR.